host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM mcr.microsoft.com/java/maven:11-zulu-debian10
+FROM mcr.microsoft.com/azure-functions/java:3.0-java11-build
```

Comment:
Is this change scoped ?  It seems different than the others. I am not very concerned with it. just wondering if it should be here

Reply:
Zulu is no longer a supported OpenJDK distribution for functions so I think that change is required as such, could perhaps be in a separate PR though?

Reply:
Thank you for bring this up. Yes, zulu is not a supported jdk for functions. For core tools v3 we didn't update it to use our own java-build image which migrate from zulu to adoptium last time. We are thinking wait for next release to update it. I can have a separate PR for it if this is preferred. Also I will attach local test results on the updated core tools images. Thanks a lot!